frontend/src/components/Accordion.js

```diff
@@ -33,6 +36,18 @@ export const AccordionItem = ({
         className={contentClasses}
         hidden={!expanded}
       >
+        {canEdit && (
+          <div className="text-right">
+            <Button
+              onClick={onNavigation}
+              unstyled
+              className="smart-hub--navigator-link flex-justify-end"
+              role="button"
+            >
+              Edit
+            </Button>
```

Comment:
Little funky that this a button instead of a link. Also, I'm not sure if we want four buttons on the page with the exact same text -> "edit," it might be better to include some aria or a hidden label to explain which page you are editing